Apps/Playground/Win32/App.cpp

```diff
@@ -100,27 +99,54 @@ namespace
             });
         });
 
+        // Initialize NativeWindow plugin.
+        auto width = static_cast<float>(rect.right - rect.left);
+        auto height = static_cast<float>(rect.bottom - rect.top);
+        runtime->Dispatch([hWnd, width, height](Napi::Env env)
+        {
+            Babylon::NativeWindow::Initialize(env, hWnd, width, height);
+        });
+
+        // Initialize NativeEngine plugin.
+        Babylon::InitializeNativeEngine(*runtime, hWnd, width, height);
+
+        // Initialize XMLHttpRequest plugin.
+        runtime->Dispatch([rootUrl](Napi::Env env)
+        {
+            Babylon::XMLHttpRequest::Initialize(env, rootUrl.data());
```

Comment:
This is dangerous and probably doesn't work at all, must have dodged tests: the char* being passed to `Initialize` points to a copy of rootUrl local to this lambda, which will get cleaned up long before any `XMLHttpRequest` objects are created. `XMLHttpRequest::Initialize` should make a copy of the text and put it somewhere safe.